src/writer/mod.rs

```diff
@@ -958,48 +954,32 @@ impl<'a> DERWriter<'a> {
         where F: FnOnce(DERWriter) -> T {
         let tag = if let Some(tag) = self.implicit_tag { tag } else { tag };
         self.implicit_tag = None;
-        let writer = DERWriter {
-            buf: self.buf,
-            implicit_tag: Some(tag),
-            override_pc: self.override_pc,
-        };
+        let mut writer = DERWriter::from_buf(self.buf);
+        writer.implicit_tag = Some(tag);
         return callback(writer);
     }
 
-    /// Overrides the primitive/constructed bit on the next value
+    /// Writes the arbitrary tagged DER value in `der`.
```

Comment:
I think the word here needs to be `arbitrarily` but I'm no native speaker.

Reply:
Both are correct but have different parse trees.
* (arbitrary (tagged (DER value)))
* ((arbitrarily tagged) (DER value))

Reply:
Thanks, it's fine then :)